### PR TITLE
Dina Bee Farm

### PR DESCRIPTION
**Pull request by vtange**
**to imc**

2 NPCs here do not talk. Probably missed quests for them if they were quest-givers.

Many bosses here show "Defeat %s"

There is a quest where the player draws oil out of barrels and collects the cork stoppers from the barrels, but the quest log says "Eliminate Cork Stoppers", based off "Eliminate %s" I think. I can't change that

### CODE REVIEW

**Comment by Sunaries** [comment]
:+1:

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -969,39 +969,39 @@ QUEST_LV_0200_20150317_000968	{memo X}Thank you for saying that. {nl}I guess it
...
-QUEST_LV_0200_20150317_000984	You brought it! {nl}I hope this will be enough to fool them.
-QUEST_LV_0200_20150317_000985	By the way, I'm worried about mister Dorjen. {nl}He's about to explode because of the monsters that's ruining his fortune. {nl}
-QUEST_LV_0200_20150317_000986	I think he is planning to burn the whole forest.. {nl}He just won't listen to us. Can you stop him?
-QUEST_LV_0200_20150317_000987	Oil barrels are spread out in Mikolas Brewery. You must get rid of those first. {nl}And mister Dorjen.. Please help us.
+QUEST_LV_0200_20150317_000983	$The monsters that smell sweet must have empty beehives. {nl}Goddess Austeja, we will need to borrow some of your help here.
+QUEST_LV_0200_20150317_000984	$You got one! {nl}I hope this will be enough to fool them.
+QUEST_LV_0200_20150317_000985	$By the way, I'm worried about the brewery's Mr. Dorjen. {nl}After seeing his life's work ruined by monsters, he's been fuming. {nl}
+QUEST_LV_0200_20150317_000986	$I think he's been planning to burn the entire forest.. {nl}And he not listening to us. Can you stop him?
```
Comment (terrapin47) [inline QUEST_LV_0200.tsv:986]: and he is not listening

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -2428,39 +2428,39 @@ QUEST_LV_0200_20150317_002427	{memo X}Knight Titas told you to help Julian squad
...
-QUEST_LV_0200_20150317_002443	Pull out the oil from the oil container
-QUEST_LV_0200_20150317_002444	Villager Darren told you that the brewer, Dorzen, may not stand the anger anymore and burn this forest with the monsters. Before you persuade Dorzen, pull out the oil from the container.
-QUEST_LV_0200_20150317_002445	Talk to Dorzen
-QUEST_LV_0200_20150317_002446	You pulled out all the oil from the oil containers that you can see. Talk to the brewer, Dorzen.
+QUEST_LV_0200_20150317_002431	$Talk to Villager Darren
+QUEST_LV_0200_20150317_002432	$Darren seems to be puzzled. Listen to him.
+QUEST_LV_0200_20150317_002433	$Check the beehives at Rododun Apiary
+QUEST_LV_0200_20150317_002434	$Darren is worried about whether his beehives at Rododun Apiary are okay after they were attacked by monsters. Go to Rododun Apiary and check whether the beehives are okay.
+QUEST_LV_0200_20150317_002435	$Fortunately, the beehives are okay. Return to Villager Darren and let him know about this.
+QUEST_LV_0200_20150317_002436	$The beehives at Rododun Apiary are okay, but you should do something on them. Talk to the relieved Villager Darren.
```
Comment (terrapin47) [inline QUEST_LV_0200.tsv:2436]: but you should do something on them -> ?  ( What is this saying? )

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -3878,7 +3878,7 @@ QUEST_LV_0200_20150323_003877	The person who is leading the mission at Dvasia Pe
 QUEST_LV_0200_20150323_003878	Thanks for your words.{nl}But it would be better to support Julian's squad at Dvasia Peak instead of this place.{nl}
 QUEST_LV_0200_20150323_003879	{memo X}I heard that the hardness and the elasticity of the wood sticks of Firent at Nepritas Cliffs is good.{nl}You can get the rope that was made from tieing those sticks from Red Pantos with spears. Please get me those first.
 QUEST_LV_0200_20150323_003880	At first, they were gathered to Uskis Arable Land and Spring Light Wood.{nl}But, not after a long time, they came to this place.{nl}They must be going after the honey.
-QUEST_LV_0200_20150323_003881	I think I can trust you more than any other Revelators.{nl}If it's okay for you, could you look for Maras at Vilna Forest up there?
+QUEST_LV_0200_20150323_003881	$You're the most trustworthy Revelator I've ever met.{nl}If it's okay for you, could you look for this guy Maras at Vilna Forest up north?
```
Comment (terrapin47) [inline QUEST_LV_0200.tsv:3881]: "if it's okay for you" -> "if it's okay with you"

File: QUEST_LV_0200.tsv [hunk 4]
```diff
@@ -969,39 +969,39 @@ QUEST_LV_0200_20150317_000968	{memo X}Thank you for saying that. {nl}I guess it
...
+QUEST_LV_0200_20150317_000993	$Calm down? Do I look like I can calm down?! {nl}Okay. Maybe I can calm down if you get my signature mead from the Honey Wine Warehouse. {nl}That's if you can get it.
+QUEST_LV_0200_20150317_000994	$I bet you won't be able to get it. {nl}Just know that if you fail, I will burn down my workshop.
+QUEST_LV_0200_20150317_000995	$Oh my, did you actually get that? {nl}I thought you would give up because of the monsters.. That's awesome man!
+QUEST_LV_0200_20150317_000996	$Alright. You win. I won't burn down my workshop as I said. {nl}But I'm still pissed. All those years of work.. because of those monsters..{nl}
+QUEST_LV_0200_20150317_000997	$Thank you very much. {nl}I will feel much better if you do that.
+QUEST_LV_0200_20150317_000998	$I won't think about burning down the forest again. {nl}I'm sure a person with abilities like yours can save our village.
+QUEST_LV_0200_20150317_000999	$That is enough. Wow, Joseph did find great talent,{nl} as much as we lost the apiary and brewery. {nl}
```
Comment (terrapin47) [inline QUEST_LV_0200.tsv:999]: "as much as we lost the apiary" -> ? ( Unsure what this is saying )
"to make up for losing the apiary and brewery."
"even though we lost the apiary and brewery."

**Comment by vtange** [comment]
Good To Merge

**Comment by terrapin47** [comment]
:+1:

**Comment by imcgames** [comment]
There are some of %s case bugs in kCBT. It seems a bug, but I'm not sure whether it will be exists on iCBT or not. I will check it when preparing the test server for 2nd iCBT.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**